Form/Type/TagType.php

```diff
@@ -4,7 +4,6 @@
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\OptionsResolver\OptionsResolverInterface;
```

Comment:
hum, probably yes to mantain the bundle compatible with Symfony version lower to 2.6

this can be the solution: http://stackoverflow.com/a/28671099/3625883